### PR TITLE
Add redirect from Heroku domain to custom domain

### LINKED ISSUE #70: Redirect users and search engines from the old domain to the new one
## What

Redirect users and search engines from the old domain to the new one

## Why

Users shouldn't get broken links because we've moved domains

## Who needs to work on this

 - developers

## Who needs to review this

 - developers

## Done when

- [x] We are able to redirect users from the old to the new domain

### PR DESCRIPTION
**Pull request by lfdebrux**
Closes #70

### CODE REVIEW

**Comment by lfdebrux** [comment]
Tested with https://prototype-kit-docs-pr-94.herokuapp.com/, by setting the config vars for that review app. Notice how it takes you to https://prototype-kit.service.gov.uk.

File: server.js [hunk 1]
```diff
@@ -108,6 +108,16 @@ utils.addCheckedFunction(nunjucksAppEnv)
 // Redirect root to /docs
```
Comment (joelanman) [inline server.js]: this comment is for the middleware below (line 121) maybe move and have a new one for this middleware?

Comment (lfdebrux) [reply server.js]: 👍 fixed

**Review by joelanman — APPROVED** [review]
thanks!

**Pull request merged**